configure.ac

```diff
@@ -191,6 +197,24 @@ AC_DEFINE([NETDATA_WITH_UUID], [1], [uuid settings])
 OPTIONAL_UUID_CLFAGS="${UUID_CFLAGS}"
 OPTIONAL_UUID_LIBS="${UUID_LIBS}"
 
+if test ! "${enable_plugin_freeipmi}" = "no"; then
+	PKG_CHECK_MODULES(
+		[IPMIMONITORING],
+		[libipmimonitoring],
+		,
+		:
```

Comment:
instead of using heuristics mark
```
PKG_CHECK_MODULES(
   ...,
   ...,
   [have_xxxx=yes],
   [have_xxxx=no]
)
if "${have_xxx}" = "yes"; then
```